ts/Core/Time.ts

```diff
@@ -410,16 +410,19 @@ class Time {
             return (timestamp: number | Date): number => {
 
                 try {
-                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-                    const [date, gmt, hours, colon, minutes = 0] =
+                    const [hours, minutes = 0] =
                         // eslint-disable-next-line new-cap
                         Intl.DateTimeFormat('en', {
                             timeZone: options.timezone,
                             timeZoneName: 'shortOffset'
                         })
                             .format(timestamp)
                             .split(/(GMT|:)/)
-                            .map(Number),
+                            .map(Number)
+                            // Filter NaNs (string values like "GMT" and ":")
+                            .filter(function (predicate): boolean {
+                                return !isNaN(predicate);
+                            }),
```

Comment:
I don't think we should add logic just to comply with shortcomings of the linter, so I am happy with the original use of `// eslint-disable-next-line` here.